### ISSUE TITLE
iOS version missing documentation

### ISSUE THREAD

**Issue by midway** [CANDY-HOUSE/SesameSDK_iOS_with_DemoApp#53]
The android repository has documentation but the ios repository does not.  Is this planned to be addressed in the near future?

**Comment by Jerming** [comment]
Yes, we do plan to address the lack of documentation for the iOS repository. We have one more product launch event planned for the remainder of this year, so we haven't had the time to release the 90% completed documentation just yet. In the meantime, the structure of the iOS version of SesameSDK is very very very similar to that of the Android version. I would recommend using the Android version's documentation as a temporary guide to familiarize yourself with the iOS version of SesameSDK.
